imagebuildah/build.go

```diff
@@ -480,7 +480,12 @@ func preprocessContainerfileContents(logger *logrus.Logger, containerfile string
 	stdoutBuffer := bytes.Buffer{}
 	stderrBuffer := bytes.Buffer{}
 
-	cmd := exec.Command(cppPath, "-E", "-iquote", ctxDir, "-traditional", "-undef", "-")
+	cppArgs := []string{"-E", "-iquote", ctxDir, "-traditional", "-undef", "-"}
+	if flags, ok := os.LookupEnv("BUILDAH_CPPFLAGS"); ok {
+		cppArgs = append(cppArgs, strings.Fields(flags)...)
```

Comment:
This should probably be using `github.com/mattn/go-shellwords.Parse()` to handle quotes in the value.